### DISCUSSION TITLE
There are not notebooks in the notebooks folder.

### DISCUSSION THREAD

**Discussion by rcyost** [TDAmeritrade/stumpy#862] [General]
Where are the tutorials?

**Comment by seanlaw** [comment]
@rcyost Thank you for your question and welcome to the STUMPY directory.

> There are not notebooks in the notebooks folder. Where are the tutorials?

In general, you can see all of the tutorials [here](https://stumpy.readthedocs.io/en/latest/tutorials.html). At the top of most of the tutorials, there is a Binder badge that will allow you to run an interactive session right in the browser. Of course, if you'd like to clone the repo and execute the tutorials locally, please look in the [docs directory](https://github.com/TDAmeritrade/stumpy/tree/main/docs)

Note that the contents of the notebooks directory are simply symbolicly linked from the docs directory (i.e., they should be identical). As stated in the README within the notebooks directory, the notebooks directory is strictly for Binder.
